docs/differences/index_template.txt

```diff
@@ -6,6 +6,105 @@ MicroPython differences from CPython
 The operations listed in this section produce conflicting results in MicroPython when compared to standard Python.
 MicroPython implements Python 3.4 and some select features of Python 3.5.
 
+Below is a list of finalised/accepted PEPs for 3.5 grouped into their impact to MicroPython.
```

Comment:
It is not really clear to me what this means; is this a list of features which MicroPython implements?